.github/ISSUE_TEMPLATE/10_package_request.yml

```
@@ -0,0 +1,31 @@
+name: "Request: Nix Package"
+description: "Package requests have been deprecated. Please open a pull request with your desired package instead."
+title: "Package Request"
+labels: ["0.kind: packaging request", "4.workflow: auto-close"]
+body:
+  - type: "markdown"
+    attributes:
+      value: |
+        <p align="center">
+          <a href="https://nixos.org">
+            <picture>
+              <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/NixOS/nixos-artwork/refs/heads/master/logo/nixos.svg">
+              <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/NixOS/nixos-artwork/refs/heads/master/logo/nixos-white.svg">
+              <img src="https://raw.githubusercontent.com/NixOS/nixos-artwork/refs/heads/master/logo/nixos.svg" width="400px" alt="NixOS logo">
+            </picture>
+          </a>
+        </p>
+
+        Thank you for your interest in packaging new software in Nixpkgs. Unfortunately, in order to mitigate the unsustainable growth of unmaintained packages, **Nixpkgs is no longer using package requests to coordinate new package additions**.
+
+        That said, **we encourage you to [contribute](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#quick-start-to-adding-a-package) this package as a Pull Request yourself** - anyone can [become a package maintainer](https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md)! Should you need any help during packaging, feel free to reach out on [Matrix](https://matrix.to/#/#dev:nixos.org) or [Discourse](https://discourse.nixos.org/c/dev/14). You are also welcome to open custom _Tracking_ issues if you would like the assistance of other volunteers in packaging a software ecosystem.
```

Do issue forms support reference-style markdown links? I personally find they make the markdown easier to read vs inline links

```suggestion
        That said, **we encourage you to [contribute] this package as a Pull Request yourself** - anyone can [become a package maintainer][maintainers]! Should you need any help during packaging, feel free to reach out on [Matrix] or [Discourse]. You are also welcome to open custom _Tracking_ issues if you would like the assistance of other volunteers in packaging a software ecosystem.
        
        [contribute]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#quick-start-to-adding-a-package
        [maintainers]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
        [Matrix]: https://matrix.to/#/#dev:nixos.org
        [Discourse]: https://discourse.nixos.org/c/dev/14
```

---

> That said, **we encourage you to [contribute] this package as a Pull Request yourself** - anyone can [become a package maintainer]! Should you need any help during packaging, feel free to reach out on [Matrix] or [Discourse].

Wording nit-picks:

I'd attempt to simplify the phrasing along the lines of:

> That said, **we encourage you to [contribute] this package yourself, via a Pull Request** — anyone can [become a package maintainer]! Should you need help, feel free to reach out on [Matrix] or [Discourse].


---

> You are also welcome to open custom _Tracking_ issues if you would like the assistance of other volunteers in packaging a software ecosystem.

This seems do imply that nixpkgs _does_ allow "package request" issues... I suspect we can find a wording that implies such issues should always be attached to WIP code, draft PRs, etc?

Also, should that sentence link to the "blank issue" template?

---

Removed the last sentence, removed the in-line links, and fixed the wording.